src/tutorials/0008-merkle-dags/02.md

```diff
@@ -0,0 +1,63 @@
+---
+    title: "Advantages of properly structured data"
+    type: "multiple-choice"
+---
+
+The choices we make when structuring our data have important
+consequences. Consider the example of a photo library, each image
+depicting different people, places, and events over the course of
+several years. We could structure this data in many ways&mdash;we
+could even elect to not structure it at all!&mdash;and each choice
+would have significant implications for our interactions with the
+library.
+
+Structure can serve as an index for data, affecting the speed
+with which we can locate and retrieve specific pictures in our
+photo library. Structure can impart semantics to data as well, by
```

Comment:
As a non-native speaker I had to lookup what `impart` means.
This would've worked better for me:

```suggestion
photo library. Structure can add semantics to data as well, by
```

Reply:
I appreciate it!